js-exercises/bouncer/bouncer.js

```diff
@@ -0,0 +1,5 @@
+function bouncer(array) {
+  return array.filter(x => x);
+}
+
```

Comment:
Hey Yash, slick seems to be your style (given the previous example :) ). If so, you could condense this further. A single line while having a slightly more meaningful variable name without affecting readability too much would also work I suppose  - 

`const bouncer = array => array.filter(item => item)`

I guess it's a matter of opinion. Good job nonetheless!